inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/transform/StreamTransformService.java

```diff
@@ -47,14 +49,31 @@ public interface StreamTransformService {
      */
     Integer save(TransformRequest request, UserInfo opInfo);
 
+    /**
+     * Query transform information based on inlong group id and inlong stream id.
+     *
+     * @param request the transform page request
+     * @return the transform response
+     */
+    PageResult<TransformResponse> listTransform(TransformPageRequest request);
+
+    /**
+     * Query transform information based on id
+     *
+     * @param id transform id.
+     * @param opInfo userinfo of operator
+     * @return transform info
+     */
+    TransformResponse get(Integer id, UserInfo opInfo);
+
     /**
      * Query transform information based on inlong group id and inlong stream id.
      *
      * @param groupId the inlong group id
      * @param streamId the inlong stream id
      * @return the transform response
      */
-    List<TransformResponse> listTransform(String groupId, String streamId);
```

Comment:
No need to change this method name and suggest changing the Page Request to `listByCondition` or `listByPagination`.

If you change to `listByPagination`, suggest changing the other interfaces alse, including StreamSource, and StreamSink.

Reply:
Done.